UITests/UITests.App/TestInterop.cs

```diff
@@ -28,7 +28,7 @@ public static void Error(string format, params object[] args)
             LogMessage("Error", "[Error] " + format, args);
         }
 
-        private static void LogMessage(string level, string format, object[] args)
+        private static async void LogMessage(string level, string format, object[] args)
         {
```

Comment:
This feels like its out of the scope of this PR.

Reply:
Do you want me to separate it? Seemed like a simple waning fix!

Reply:
It does not have to be separated, but if its was on its own or with other similar changes then it would the reviewers don't need to think of anything else to merge it. I general if its small and not related at all to anything else in the PR it makes it even better as a separate PR. Once again I don't think its necessary.

Reply:
On another note about this change, while this gets rid of the warning. It leaves me wondering if we should make the method return a Task? Other it being considered good practice to avoid `async void` methods, it feels like making it `async void` only hides the same warning from the callers of `LogMessage` where if it returned a `Task` then you'd see the same warning at its various call sites.

Reply:
I think in this case, its A-OK! Since, this method doesn't need to return to the context. Anyway, I made a few changes and so, I decided to separate the PR. We'll continue the discussion there.

Reply:
Separated into a new PR #4125